extensions/wikia/RTE/js/RTE.js

```diff
@@ -17,7 +17,13 @@
 			baseFloatZIndex: 5000101, // $zTop + 1 from _layout.scss
 			bodyClass: 'WikiaArticle',
 			bodyId: 'bodyContent',
-			contentsCss: [$.getSassLocalURL('extensions/wikia/RTE/css/content.scss'), window.RTESiteCss],
+			contentsCss: [
+				$.getSassLocalURL('extensions/wikia/RTE/css/content.scss'),
+				$.getSassLocalURL('extensions/wikia/PortableInfobox/styles/PortableInfobox.scss'),
+				// TODO check $wgEnablePortableInfoboxEuropaTheme first
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)